### ISSUE TITLE
Translate documentation

### ISSUE THREAD

**Issue by pierreozoux** [nextcloud/desktop#3234]
It looks like it is currently not translated.

Would you be interested to have the doc translated?

**Comment by dhebert91** [comment]
Thank's Pierre for this issue. I'd like to contribute to the translation of this documentation : https://docs.nextcloud.com/desktop/2.6/

**Comment by FlexW** [comment]
I don't think it makes sense to translate https://docs.nextcloud.com/desktop/2.6/ as this an old version. The newest documentation can be found here https://docs.nextcloud.com/desktop/3.2. Having that said, I think it would be great to have translations for the documentation. However, I'm unsure how this can be achieved in an efficient way. Could we use Transifex for that? I think this would require some work from our side to change the infrastructure? @mgallien @er-vin  @Valdnet

**Comment by Valdnet** [comment]
> Could we use Transifex for that?

By all means, documentation should be added to Transifex.

User documentation for the Nextcloud instance already exists in Transifex and is translated 100% in three languages.

**Comment by p-bo** [comment]
Any news regarding possibility to translate desktop client documentation please? Is it possible to use the same mechanism as in case of https://github.com/nextcloud/documentation/tree/master/user_manual/locale (thus what is remaining is to adapt the same recopies and connect to newly created subproject on Transifex) , or is there some difference and completely different approach is needed?

**Comment by p-bo** [comment]
@FlexW Just curiosity (all in positive and constructive mindset) – why this attitude whether it is worth or even wanted?

Localization (of user interface and documentation) is one of important pieces to cope with digital divide (and FLOSS software is serving there very well due its spirit) - not everyone had / has / will have luck to learn English (older users, who were historically forced to learn some other language(s); then people from disadvantaged, undeserved communities, etc.) I do understand, that it is difficult to have empathy with others, when one didn't have experience with their problems (is fluent in English), but please imagine situation, when your language skills are outdated (and irreplaceable, because your mental capabilities are affected by age) or non-existent (you haven't access to education of sufficient quality) and that state is not result of your own fault…

P.S.: as you see, human resources to implement this and maintain translations volunteered already - thus no additional burden for you ;-)

**Comment by Valdnet** [comment]
In my opinion, it's nice to read in your native language. It takes care of users (the older ones too) and adds seriousness to the application.

**Comment by tobiasKaminsky** [comment]
> @tobiasKaminsky @mgallien @allexzander @camilasan What do you think? Is that a task worth putting on the board? Or do we even want that?

We can help with setting it up. Translation is done by our awesome community 👍

**Comment by tobiasKaminsky** [comment]
> As I did the plumbing for the user manual, I could do it also. I think it is a one day work.
> 
> Let me know :)

Ah, cool, that you want to help :-)
Please do it 💙

If you need any help, e.g. in creating new resources in Transifex just write here.

**Comment by pierreozoux** [comment]
@tobiasKaminsky are there any good reasons of not having this inside the doc repo directly?

Doing it for the user manual was already tedious, and it is still not over.

I think I'd prefer to have it there as well, I think it makes more sense.

Then the project would be to move this repo there as a subfolder. If you are ok with the idea, I could do the work as well.

Let me know.

**Comment by tobiasKaminsky** [comment]
As discussed with @AndyScherzinger it is okay to move this to doc repo.
If you want to do this, this would be awesome! :tada: 
If you need any help, please just ping here.

**Comment by p-bo** [comment]
@pierreozoux Hi, it looks regarding this? There is another request for this: https://github.com/nextcloud/documentation/issues/7729

**Comment by rakekniven** [comment]
As I wrote in linked issue:

_As the client is a very popular part of Nextcloud and there are plenty of questions regarding it at the forums it would help to have it localised._

Please move content to documentation repo and add it to Transifex.

**Comment by mgallien** [comment]
I am under the impression that adding client documentation in the server documentation repository would lead to complication in particular to be able to align desktop client and matching documentation in a repository where branches currently are used to do that aligned with server releases
I can be proven wrong but I feel like it would probably be easier to just sync the client documentation to transifex like it is done for server documentation

**Comment by pierreozoux** [comment]
Ok, I was thinking to move this folder:
https://github.com/nextcloud/desktop/tree/master/doc
here:
https://github.com/nextcloud/documentation/tree/master/user_manual

But yeah, it is versioned with server version...

I think you are right @mgallien as I did the  other automation, I'll copy it here. I'll start today.

**Comment by mgallien** [comment]
> Ok, I was thinking to move this folder: https://github.com/nextcloud/desktop/tree/master/doc here: https://github.com/nextcloud/documentation/tree/master/user_manual
> 
> But yeah, it is versioned with server version...
> 
> I think you are right @mgallien as I did the other automation, I'll copy it here. I'll start today.

Thanks a lot @pierreozoux

**Comment by p-bo** [comment]
@pierreozoux Thanks for your dedication!

**Comment by rakekniven** [comment]
Hello, 

good point about the versioning. I will close my issue in docu repo.

Another point is the style of the client docs. They are display differently to the server. 
Both use Sphinx, or?
Can we align the style?

In the future there will be more docs. I am planning to write something for the iOS app and for sure I will use sphinx as well.

**Comment by p-bo** [comment]
@rakekniven Sorry for hijacking this thread a bit, but it would be great to resurrect docs for Android app too - see https://github.com/nextcloud/documentation/issues/1583 - in any case noble idea to improve situation with non-existent docs to mobile client(s)!

**Comment by rakekniven** [comment]
> but it would be great to resurrect docs for Android app too

Hello @p-bo Happy to have you on board. Feel free to start.

**Comment by p-bo** [comment]
Just for reference - @pierreozoux created/fixed localization synchronizing/merging to/from Transifex in doc repo (for server user docs) and hopefully will port this solution also into this repo for client docs. Really appreciated!

**Comment by rakekniven** [comment]
> Really appreciated!

Absolutely. 🚀

**Comment by pierreozoux** [comment]
@p-bo sorry it takes so long..
I'm actually still not happy there..
and this is not something I can test on my machine :/
I really want to fix it for good there, and then I just need to copy here.
hopefully next week it will be good :)

**Comment by pierreozoux** [comment]
@rakekniven https://github.com/nextcloud/documentation/pull/8498#issuecomment-1291804016

**Comment by mgallien** [comment]
@pierreozoux @rakekniven 
status of this is unclear to me
do we have any technical issues blocking this ?

**Comment by pierreozoux** [comment]
@mgallien @rakekniven my thinking around this topic is still the same and is documented [here](https://github.com/nextcloud/documentation/pull/8498#issuecomment-1291804016) indeed :)

And I think we should normalize all Nc projects to follow same approach.

In the coming weeks, I'd have an hour to discuss about this during a visio if somebody from Nc wants and agree with all stake holders that this is our wish.
And about the proposition to do it, I'll be able to do it in April or May, there is roughly a week of work I think.

**Comment by DaphneMuller** [comment]
@pierreozoux @rakekniven just FYI @mgallien will reach out to you beginning of next week to schedule a call to align next steps :)

**Comment by mgallien** [comment]
@rakekniven sent a direct mail to @pierreozoux and @skjnldsv 
feel free to told me if you would still like to be involved

**Comment by skjnldsv** [comment]
> I think @skjnldsv will be the right person as he has been part of the discussion tech side.

This has been a long time :see_no_evil:

**Comment by skjnldsv** [comment]
I merged and updated some workflows.
Just manually triggerred an update, I'll update this thread once it's rolling again :hourglass_flowing_sand:

**Comment by skjnldsv** [comment]
Automated process fix is in https://github.com/nextcloud/documentation/pull/9669
Please review and approve

Then we can merge and trigger the automated update again.

**Comment by skjnldsv** [comment]
> Automated process fix is in [nextcloud/documentation#9669](https://github.com/nextcloud/documentation/pull/9669) Please review and approve
> 
> Then we can merge and trigger the automated update again.

Merged, triggered and fully functional with auto merge! :rocket: 
Feel free to use the same process for this documentation so everyone can translate.
Once this is setup on this repo, I can help configuring transifex for this. It's been a while but I can.

For this repository, I see a few issues already:
1. The theme used is still the old nextcloud one, we moved to readthedoc long time ago
  https://github.com/nextcloud/documentation/blob/ef5f563a259e0c07e32a11f6dc90a5148281bda7/requirements.txt#LL4C26-L4C26 
2. Then we have the lang local config
  https://github.com/nextcloud/documentation/blob/ef5f563a259e0c07e32a11f6dc90a5148281bda7/user_manual/conf.py#L317
   https://github.com/nextcloud/documentation/commit/81e8a66c620b7daa847c9e1e069aa549a72e9657

This is not a tiny amount of work though https://github.com/nextcloud/documentation/pull/2549

**Comment by p-bo** [comment]
Hello @pierreozoux,

hope you are doing well. Please, are there any news regarding this please?

Have a nice day :-)

**Comment by Rello** [comment]
@rakekniven 

the docu was moved to the main repo. I guess it will be now automatically part of the existing translation, correct?

https://github.com/nextcloud/documentation

**Comment by mgallien** [comment]
@rakekniven I did a quick check
I confirm that there are no desktop files client related resources apparently
I checked the cron job syncing the translation and it seems to be working fine without errors (at least visible ones)
I also saw an existing project for desktop files client single documentation https://app.transifex.com/nextcloud/nextcloud-user-documentation/content/
I guess one should remove it

I right now have no idea yet how to help

**Comment by Rello** [comment]
@rakekniven 
there was a problem with the action, which did not run for a long time. 
there will be many updates coming through with the next run

https://github.com/nextcloud/documentation/pull/12611

we can finally close this. if not, get back here.

thank you!

**Comment by Rello** [comment]
> > there will be many updates coming through with the next run
> 
> Wow, now I'm curious to see what will be on Transifex tomorrow :-)

get up early *g*
